2024-04-02 17:50:49,698 INFO    MainThread:252512 [wandb_setup.py:_flush():75] Loading settings from /home/presta/.config/wandb/settings
2024-04-02 17:50:49,698 INFO    MainThread:252512 [wandb_setup.py:_flush():75] Loading settings from /scratch/DVC-PyTorch/wandb/settings
2024-04-02 17:50:49,698 INFO    MainThread:252512 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2024-04-02 17:50:49,698 INFO    MainThread:252512 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'main_dvc.py', 'program': 'main_dvc.py'}
2024-04-02 17:50:49,730 INFO    MainThread:252512 [wandb_init.py:_log_setup():386] Logging user logs to /scratch/DVC-PyTorch/wandb/run-20240402_175049-1j4i2j4q/logs/debug.log
2024-04-02 17:50:49,731 INFO    MainThread:252512 [wandb_init.py:_log_setup():387] Logging internal logs to /scratch/DVC-PyTorch/wandb/run-20240402_175049-1j4i2j4q/logs/debug-internal.log
2024-04-02 17:50:49,731 INFO    MainThread:252512 [wandb_init.py:init():420] calling init triggers
2024-04-02 17:50:49,731 INFO    MainThread:252512 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'log': '', 'pretrain': '', 'test': False, 'testhevc': False, 'testvtl': False, 'testmcl': False, 'testauc': False, 'rerank': False, 'allpick': False, 'config': 'DVC_1024.json'}
2024-04-02 17:50:49,731 INFO    MainThread:252512 [wandb_init.py:init():471] starting backend
2024-04-02 17:50:49,732 INFO    MainThread:252512 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-02 17:50:49,755 INFO    MainThread:252512 [backend.py:ensure_launched():219] starting backend process...
2024-04-02 17:50:49,767 INFO    MainThread:252512 [backend.py:ensure_launched():225] started backend process with pid: 252579
2024-04-02 17:50:49,768 INFO    MainThread:252512 [wandb_init.py:init():480] backend started and connected
2024-04-02 17:50:49,773 INFO    MainThread:252512 [wandb_init.py:init():550] updated telemetry
2024-04-02 17:50:49,947 INFO    MainThread:252512 [wandb_init.py:init():581] communicating current version
2024-04-02 17:50:50,794 INFO    MainThread:252512 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.16.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-02 17:50:50,794 INFO    MainThread:252512 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2024-04-02 17:50:51,026 INFO    MainThread:252512 [wandb_init.py:init():624] starting run threads in backend
2024-04-02 17:50:51,871 INFO    MainThread:252512 [wandb_run.py:_console_start():1827] atexit reg
2024-04-02 17:50:51,874 INFO    MainThread:252512 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2024-04-02 17:50:51,874 INFO    MainThread:252512 [wandb_run.py:_redirect():1706] Redirecting console.
2024-04-02 17:50:51,877 INFO    MainThread:252512 [wandb_run.py:_redirect():1762] Redirects installed.
2024-04-02 17:50:51,877 INFO    MainThread:252512 [wandb_init.py:init():651] run started, returning control to user process
2024-04-02 17:50:55,075 INFO    MainThread:252512 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2024-04-02 17:50:55,079 INFO    MainThread:252512 [wandb_run.py:_restore():1769] restore
2024-04-02 17:50:57,551 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2024-04-02 17:50:57,797 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2024-04-02 17:50:58,117 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2024-04-02 17:50:59,231 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 4088
}

2024-04-02 17:50:59,333 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 4088
}

2024-04-02 17:50:59,439 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 4088
}

2024-04-02 17:50:59,541 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2024-04-02 17:50:59,643 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2024-04-02 17:50:59,747 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2024-04-02 17:50:59,850 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2024-04-02 17:51:00,094 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2024-04-02 17:51:00,527 INFO    MainThread:252512 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}
local_info {
}

2024-04-02 17:51:01,661 INFO    MainThread:252512 [wandb_run.py:_append_files():2194] logging synced files
